src/GT.UI/Views/Listing/_ListingFilterPartial.cshtml

```diff
@@ -1,25 +1,58 @@
-﻿@model GT.Core.FilterModels.Impl.ListingFilterModel
```

Comment:
For future improvment consideration making a bootstrap collapsable over the filter partial, see: https://getbootstrap.com/docs/4.0/components/collapse/